ivadomed/postprocessing.py

```diff
@@ -137,3 +138,47 @@ def mask_predictions(predictions, mask_binary):
     assert predictions.shape == mask_binary.shape
     assert np.array_equal(mask_binary, mask_binary.astype(bool))
     return predictions * mask_binary
+
+
+def coordinate_from_heatmap(nifti_image, thresh=0.3):
+    """
+    Retrieve coordinates of local maxima in a soft segmentation.
+    Args:
+        nifti_image (nibabel object): nifti image of the soft output.
+        thresh (float): Relative threshold for local maxima, i.e., after normalizing
+        the min and max between 0 and 1, respectively.
+
+    Returns:
+        list: A list of computed coordinates found by local maximum. each element will be a list composed of
+        [x, y, z]
+    """
+    image_ref = nib.load(nifti_image)
```

Comment:
Is nifti_image a nibabel object or a fname? I am confused by the nib.load

Reply:
It' a fname so i'll update this.

Reply:
So It was resolved when I switched to nibabel object.